.dockerignore

```
@@ -35,3 +35,7 @@ token-tracker/
 # Legacy V1 contracts bindings.
 # We won't generate new bindings in the docker build process, but use the existing ones.
 !pkg/chain/gen
+
+pkg/chain/ethereum/threshold/gen/_address/TokenStaking
+pkg/chain/ethereum/beacon/gen/_address/RandomBeacon
+pkg/chain/ethereum/ecdsa/gen/_address/WalletRegistry
```

Could the rule be more general?

E.g.
```suggestion
**/gen/_address
```
Would this work fine?

---

sure, `**/gen/_address` won't work, but `**/gen/_address/*` will. [2e24cca](https://github.com/keep-network/keep-core/pull/3410/commits/2e24cca713dc3d508669376918c4bda602859bf0)